ortholog-mapping/README.md

```diff
@@ -1,12 +1,12 @@
 # <u>refine.bio Examples: Ortholog Mapping </u>
-This directory contains example workflows of how to map data from one species 
+This directory contains example workflows of how to map data from one species
 onto the orthologous genes from another species using data from
 <a href="refine.bio.org"> refine.bio</a>.
 
 ### Ortholog Mapping in R
 This <a href="https://github.com/AlexsLemonade/refinebio-examples/blob/master/ortholog-mapping/ortholog_mapping_example.Rmd">
-notebook </a> illustrates how to obtain the orthologous genes of one species to 
+notebook </a> illustrates how to obtain the orthologous genes of one species to
```

Comment:
If you're updating the other READMEs to be a bit more "markdown-y," let's change this to the `[]()` format as well.